woocommerce/woocommerce-android
# [Customer Search] Handle older 800 versions of WC where "all" param is not supported as search

Closes: #9353


### Description

The PR adds handling of the older (<8.0.0) versions of the WC core, where customer search doesn't have `searby=all` parameter

### Testing instructions

* Start order creation
* Add a customer
* On the sites with version of WC 8.0.0+ (use my site for testing), notice that 4 search params are in place
* On the sites with version of WC <8.0.0+ 3 search params are in place

### Images/gif

<img width="335" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/1200c84b-9b5e-4cc8-8623-f4a993d6f9cf">
<img width="334" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/e2c5373e-e6bc-4dbd-b4b9-6e4d03486136">
